forge/test/operators/pytorch/eltwise_binary/test_pytorch_binary.py

```diff
@@ -0,0 +1,504 @@
+        "floor_divide",             #07                         - AssertionError: Encountered unsupported op types. Check error logs for more details
+        "fmod",                     #08                         - AssertionError: Encountered unsupported op types. Check error logs for more details
+        "logaddexp",                #09                         - NotImplementedError: The following operators are not implemented: ['aten::logaddexp']
+        "logaddexp2",               #10                         - NotImplementedError: The following operators are not implemented: ['aten::logaddexp2']
+        "nextafter",                #11                         - NotImplementedError: The following operators are not implemented: ['aten::nextafter']
+        "remainder",                #12                         - AssertionError: Encountered unsupported op types. Check error logs for more details
+        "fmax",                     #13                         - NotImplementedError: The following operators are not implemented: ['aten::fmax']
+        "fmin",                     #14                         - NotImplementedError: The following operators are not implemented: ['aten::fmin']
+        
+        "eq",                       #15                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: equal
+        "ne",                       #16                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: not_equal
+        "le",                       #17                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: less_equal
+        # "greater",                #18    - Alias for gt.      E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: greater
+        "gt",                       #19                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: greater
+        # "less_equal",             #20    - Alias for le.      E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: less_equal
+        "lt",                       #21                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: less
+        # "less",                   #22    - Alias for lt.      E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: less
+        "maximum",                  #23                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: maximum
+        "minimum",                  #24                         E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: minimum
+        # "not_equal",              #25    - Alias for ne.      E       RuntimeError: Unsupported operation for lowering from TTForge to TTIR: not_equal
+    ]
+
```

Comment:
78 xfailed, 10 xpassed, 2 warnings in 29.77s

There are now 10 xpassed tests in not implemented group which means new operators included.

Actually these 10 xpassed are 10 operators passing const eval pass